### ISSUE TITLE
cuda out of memory

### ISSUE THREAD

**Issue by BlarkLee** [mileyan/pseudo-LiDAR_e2e#9]
Can you please tell me what computing power (devices) and how much memories you are using for training e2dPointRCNN? I'm using two 1080 Ti with 11178 M for each gpu, but still facing the issue of "cuda out of memory" except when I use the training mode "rpn".

**Comment by rui1996** [comment]
Hi we indicate here: https://github.com/mileyan/pseudo-LiDAR_e2e/tree/master/PointRCNN#run-pointrcnn_pl_end2end-training-and-evaluation we use two RTX. I think it would be OOM for 1080Ti and 2080Ti.

**Comment by BlarkLee** [comment]
I see! Thank you !

**Comment by wanghangege** [comment]
> Can you please tell me what computing power (devices) and how much memories you are using for training e2dPointRCNN? I'm using two 1080 Ti with 11178 M for each gpu, but still facing the issue of "cuda out of memory" except when I use the training mode "rpn".

hello, I have met the same problem with you, and I have a 3060 GPU with 12GB, I trained the pointrcnn by batch_size=1, but failed. 
`RuntimeError: CUDA out of memory. Tried to allocate 128.00 MiB (GPU 0; 11.76 GiB total capacity; 9.30 GiB already allocated; 37.19 MiB free; 9.59 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
`

**Comment by wanghangege** [comment]
> Can you please tell me what computing power (devices) and how much memories you are using for training e2dPointRCNN? I'm using two 1080 Ti with 11178 M for each gpu, but still facing the issue of "cuda out of memory" except when I use the training mode "rpn".

hello, I have met the same problem with you, and I have a 3060 GPU with 12GB, I trained the pointrcnn by batch_size=1, but failed. 
`RuntimeError: CUDA out of memory. Tried to allocate 128.00 MiB (GPU 0; 11.76 GiB total capacity; 9.30 GiB already allocated; 37.19 MiB free; 9.59 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
`
